language/German/strings.po

```diff
@@ -1229,11 +1204,11 @@ msgstr "Benutzeroberfläche:   Aktiviert"
 
 #: src/emu/ui/ui.cpp:1595 src/emu/ui/ui.cpp:1605
 msgid "**Use ScrLock to toggle**"
-msgstr "** Verwenden Sie ScrLock zum umschalten**"
+msgstr "** Verwenden Sie ScrLock zum Umschalten**"
 
 #: src/emu/ui/ui.cpp:1602
 msgid "Mode: FULL Emulation"
-msgstr "Modus: KOMPLETTE Emulation"
+msgstr "Modus: VOLLSTÄNDIGE Emulation"
 
 #: src/emu/ui/ui.cpp:1603
 msgid "UI:   Disabled"
```

Comment:
msgstr "Dauerfeuer kann nicht aktiviert werden"
Ich denke so werden es alle in erinnerung haben.